manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md

```diff
@@ -165,38 +309,179 @@ PUT test-index
+      "index": ".ds-logs-elastic_agent.filebeat-default-2025.06.04-000001",
+      "managed": true,
+      "policy": "logs",
+      "index_creation_date_millis": 1749060710541,
+      "time_since_index_creation": "19.05d",
+      "lifecycle_date_millis": 1749060710541,
+      "age": "19.05d",
+      "phase": "hot",
+      "phase_time_millis": 1749060711038,
+      "action": "rollover",
+      "action_time_millis": 1749060712038,
+      "step": "check-rollover-ready",
+      "step_time_millis": 1749060712038,
+      "phase_execution": {
+        "policy": "logs",
+        "phase_definition": {
+          "min_age": "0ms",
+          "actions": {
+            "rollover": {
+              "max_age": "30d",
+              "max_primary_shard_docs": 200000000,
+              "min_docs": 1,
+              "max_primary_shard_size": "50gb"
+            }
+          }
+        },
+        "version": 1,
+        "modified_date_in_millis": 1749059754275
+      }
     }
   }
 }
 ```
 
-1. Updates all indices with names that start with `mylogs-pre-ilm`
+You can also call this API for a data stream:
 
+```console
+GET my-datastream/_ilm/explain
+```
 
+When called for a data stream, the API retrieves the current lifecycle status for the stream's backing indices:
 
-### Switch lifecycle policies [switch-lifecycle-policies]
+```json
+{
+  "indices": {
+    ".ds-logs-elastic_agent.filebeat-default-2025.06.04-000001": {
+      "index": ".ds-logs-elastic_agent.filebeat-default-2025.06.04-000001",
+      "managed": true,
+      "policy": "logs",
+      "index_creation_date_millis": 1749060710541,
+      "time_since_index_creation": "19.06d",
+      "lifecycle_date_millis": 1749060710541,
+      "age": "19.06d",
+      "phase": "hot",
+      "phase_time_millis": 1749060711038,
+      "action": "rollover",
+      "action_time_millis": 1749060712038,
+      "step": "check-rollover-ready",
+      "step_time_millis": 1749060712038,
+      "phase_execution": {
+        "policy": "logs",
+        "phase_definition": {
+          "min_age": "0ms",
+          "actions": {
+            "rollover": {
+              "max_age": "30d",
+              "max_primary_shard_docs": 200000000,
+              "min_docs": 1,
+              "max_primary_shard_size": "50gb"
+            }
+          }
+        },
+        "version": 1,
+        "modified_date_in_millis": 1749059754275
+      }
+    }
+  }
+}
+```
+:::
+::::
+
+## Switch lifecycle policies [switch-lifecycle-policies]
+
+You may want to change the ILM policy applied to an index. For example, if you're ingesting a large set of log data you may want to apply a different policy to a subset of indices containing only the most critical logs.
+
+:::::{warning}
+Be careful when changing either the `logs@lifecycle` or `metrics@lifecycle` policies as these typically manage many indices. In {{kib}}, the **Index Lifecycle Policies** table shows the number of indices currently associated with each policy.
+:::::
```

Comment:
This is a bit misleading. If I am not mistaken these are managed policies, meaning they are shipped along with elasticsearch. The recommendation for such policies is that the user should not change them, ever. If a user changes them, there are no guarantees that a future upgrade will not overwrite them. In general we recommend to create a new policy and associate them with the intended index templates or the index they want.

Reply:
Perfect. I've changed this to:

<img width="700" alt="screen6" src="https://github.com/user-attachments/assets/e809ae48-b4b6-4ddb-be57-84db0f8b0741" />

(see [here](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/1906/manage-data/lifecycle/index-lifecycle-management/policy-updates#switch-lifecycle-policies))

Reply:
Should we add a course of action to ensure the user can recover from removing a policy? Although it's not easy to anticipate everything that could go wrong.

Reply:
> a course of action to ensure the user can recover from removing a policy

I think that's something we could add to the Troubleshooting section of the docs, and then have a link from this page. If that's something we should write up I'd open a separate issue for it rather than tackle it in this PR.